CHANGELOG.md

```diff
@@ -1,11 +1,16 @@
-v1.0.9
-- Updated Readme to go from Powershell based install to Manual Install With More Detailed Instructions
-- Removed unneeded config items
-- Fixed Meta Data Sync Issue
-- For Patch Installation for V1.0.9, see bottom of Readme File
+v1.0.11
```

Comment:
1.0.10 has not been finalized. This changelog entry belongs with v1.0.10

Reply:
Looks good with [latest commit](https://github.com/Keyfactor/cscglobal-cagateway/commit/d96ef449d6d95dabc026789a96e1f701b99b5807)